msb-develop.sh

```diff
@@ -9,13 +9,7 @@ fi
 
 
 COMPOSE_FILE_LOC=docker/docker-compose.yml
```

Comment:
this is the wrong compose file for messybrainz. In fact, I can't even see a compose file any more, I suspect it disappeared when the repos were merged. Is there a reason to start up messybrainz as a standalone server?

I recommend that we delete this file entirely, and optionally add back docker-compose.yml into the messybrainz/ directory. There's no reason to have a develop.sh file if docker-compose.yml is named the default name and in a directory that can be automatically used as the project name

Reply:
Indeed, this is wrong. I am not sure whether we need messybrainz as separate server. Maybe @mayhem has some thoughts about it?

Reply:
Removed. It is still possible to run a local development server using `msb_manage.py` in case needed.

Reply:
MessyBrainz used to run as an external server and LB connected over http to submit data, but this no longer happens. I'm happy to remove this functionality and add it back later if we need it.

Reply:
Removed that as well.